integration_tests/integration/searchForABooking.cy.ts

```diff
@@ -75,14 +77,9 @@ context('Search for a booking by reference', () => {
     visitDetailsPage.prisonerName().contains('Smith, John')
   })
 
-  it('Should search via prisonerId, than navigate to the summary page', () => {
+  it.skip('Should search via prisonerId, than navigate to the summary page', () => {
```

Comment:
Did you mean to skip?

Reply:
Yes, going to be doing the integration tests for this in a later commit as it's a change of path completely in this use case. Was important to get this work testable so Anu can start with her work on this